backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/IGitMacheteCommit.java

```diff
@@ -1,7 +1,11 @@
 package com.virtuslab.gitmachete.backend.api;
 
+import java.util.Date;
+
 public interface IGitMacheteCommit {
   String getMessage();
 
   String getHash();
+
+  Date getCommitDate();
```

Comment:
Pls replace with `ZonedDateTime` + globally forbid usage of `java.util.Date` via `Regexp` (https://stackoverflow.com/questions/54911481/whats-the-difference-between-java-util-date-and-zoneddatetime)

Reply:
Or possibly timezone-less `java.util.Instant` instead of `ZonedDateTime`